teams/rustdoc-frontend.toml

```
@@ -0,0 +1,31 @@
+name = "rustdoc-frontend"
+subteam-of = "rustdoc"
+
+[people]
+leads = []
+members = [
+    "notriddle",
+    "GuillaumeGomez",
+    "jsha",
+]
+alumni = []
+
+[permissions]
+bors.rust.review = true
+dev-desktop = true
+perf = true
+
+[[github]]
+orgs = ["rust-lang", "rust-lang-nursery"]
+extra-teams = ["rustdoc"]
+
+[website]
+name = "Rustdoc web frontend"
+description = "Rustdoc frontend design, UI, UX and development"
+zulip-stream = "t-rustdoc"
+
+[rfcbot]
+label = "T-rustdoc-frontend"
+name = "Rustdoc frontend"
+ping = "rust-lang/rustdoc"
```

I think it should be this instead?

```suggestion
ping = "rust-lang/rustdoc-frontend"
```

---

It should indeed!